sympy/matrices/sparse.py

```diff
@@ -334,6 +334,17 @@ def _eval_conjugate(self):
             conj._smat[key] = value.conjugate()
         return conj
 
+    def as_real_imag(self):
+        """Returns tuple containing (real , imaginary) part of sparse matrix"""
+        from sympy.functions.elementary.complexes import re, im
+        real_smat = im_smat = self.copy()
```

Comment:
Oops ! I don't know how it got overlooked .
Now this and corresponding tests have been fixed as well.